kitsune/wiki/views.py

```diff
@@ -525,6 +525,20 @@ def review_revision(request, document_slug, revision_id):
     revision_contributors = list(set(
         based_on_revs.values_list('creator__username', flat=True)))
 
+    # Get Unreviewed Revisions
+    unreviewed_revision = list(Revision.objects.filter(
+            document=doc,
+            is_approved=False).order_by('-created'))
+    try:
+        unreviewed_revision
+    except IndexError:
+        unreviewed_revision = ''
+
+    # Remove the revision which is reviewing and get last 5 revision
+    if rev in unreviewed_revision:
+        unreviewed_revision.remove(rev)
+    unreviewed_revision = unreviewed_revision[0:5]
```

Comment:
When slicing, the first argument defaults to 0. In other words, this is better written as `unreviewed_revision = unreviewed_revision[:5]`.